src/main/java/frc/robot/subsystems/AlgaeShooter/AlgaeShooter.java

```diff
@@ -0,0 +1,25 @@
+package frc.robot.subsystems.AlgaeShooter;
+
+import org.littletonrobotics.junction.Logger;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class AlgaeShooter extends SubsystemBase{
+    private final AlgaeShooterIO io;
+    private final AlgaeShooterIOInputsAutoLogged inputs = new AlgaeShooterIOInputsAutoLogged();
+
+    //Creates AlgaeShooter
+    public AlgaeShooter(AlgaeShooterIO io) {
+        this.io = io;
+    }
+
+    public void setDutyCycle (double dutyCycle){
+        io.setDutyCycle(dutyCycle);
+    }
+
+    @Override
+    public void periodic(){
+        io.updateInputs(inputs);
+        Logger.processInputs("Algae Shooter", inputs);
+    }
```

Comment:
I also want in class commands that tell it to 
run the shooter at a given duty cycle, 
at a given velocity, 
and that wait until the shooter is at the setpoint velocity

This is so we can take this to the practice bot ASAP